app/javascript/components/editor/Header.tsx

```diff
@@ -50,5 +51,5 @@ Header.ActionKeyboardShortcuts = forwardRef<
 })
 
 Header.ActionSettings = Settings
-
-Header.ActionMore = ActionMore
+Header.ActionMore = More
```

Comment:
I renamed the component from `ActionMore` to `More` to better fit with the naming of the other action components.